lib/SPIRV/SPIRVReader.cpp

```diff
@@ -800,16 +800,27 @@ bool SPIRVToLLVM::postProcessOCLBuiltinWithArrayArguments(
   return true;
 }
 
-static char getTypeSuffix(Type *T) {
-  char Suffix;
+static std::string getTypeSuffix(Type *T, SPIRVType *SpirT) {
```

Comment:
Can we please do not mix SPIR and SPIR-V. It might be confusing.

Reply:
Agreed sorry, will fix